plv8_func.cc

```diff
@@ -186,15 +178,6 @@ SubTranBlock::exit(bool success)
 		ReleaseCurrentSubTransaction();
 	else
 		RollbackAndReleaseCurrentSubTransaction();
-
-	MemoryContextSwitchTo(m_mcontext);
-	CurrentResourceOwner = m_resowner;
-
-	/*
-	 * AtEOSubXact_SPI() should not have popped any SPI context, but just
-	 * in case it did, make sure we remain connected.
-	 */
-	SPI_restore_connection();
```

Comment:
i have most of the travis environments passing, with the exception of:

`g++: error: unrecognized command line option ‘-Wshorten-64-to-32’` - i need to figure out why the compile on that platform is not correctly figuring out the environment in `gyp`.

i also need to get a windows environment up and running (will work on that later today or sunday) - i need to make sure everything will work in windows.